index.html

```diff
@@ -128,8 +230,10 @@ <h1 class="text-4xl" id="Intro">
     </div>
     <!-- links -->
     <footer class="flex w-full mt-auto mt-2 justify-center sm:justify-end absolute sm:bottom-0 pb-1 py-3">
-      <span class="hidden sm:block absolute left-0 font-light">Developed By
-        <a class="font-light text-amber-500" href="https://github.com/FixxooXD" target="_blank">FixxooXD</a></span>
+      <!-- hidden sm:block absolute left-0 font-light -->
```

Comment:
@vedamruta why commented out this line too ?

Reply:
Actually, this was earlier used class name which I have modified but I have kept it for the maintainer to catch this change since he knows what he had earlier given as the class name.